package/rules/module.sass.js

```diff
@@ -1,8 +1,14 @@
 const getStyleRule = require('../utils/get_style_rule')
+const { resolved_paths: includePaths } = require('../config')
 
 module.exports = getStyleRule(/\.(scss|sass)$/i, true, [
   {
     loader: 'sass-loader',
-    options: { sourceMap: true }
+    options: {
+      sourceMap: true,
+      sassOptions: {
+        includePaths
+      }
```

Comment:
I believe this means that we should go for sass-loader ^8.0.0, WDYT?

https://github.com/webpack-contrib/sass-loader/blob/master/CHANGELOG.md#-breaking-changes

Previously:
https://github.com/webpack-contrib/sass-loader/tree/v7.3.1#options
Current:
https://github.com/webpack-contrib/sass-loader/tree/v8.0.2#object

Reply:
Ahh I see. Yeah, lets do that. 

I am thinking to try and upgrade master to webpack 5 beta too and see if everything works as intended but in another pr.